### PR TITLE
Delete blood pressure measurements without a patient when deleting data from other sync groups

### PR DESCRIPTION
**Pull request by vinaysshenoy**
https://app.clubhouse.io/simpledotorg/story/1413/regularly-purge-data-outside-the-current-sync-group

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/AppDatabase.kt [hunk 1]
```diff
@@ -217,6 +217,7 @@ abstract class AppDatabase : RoomDatabase() {
       val facilityIdsInCurrentSyncGroup = facilityDao().facilityIdsInSyncGroup(currentFacility.syncGroup)
 
       patientDao().deletePatientsNotInFacilities(facilityIdsInCurrentSyncGroup)
+      bloodPressureDao().deleteWithoutLinkedPatient()
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/AppDatabase.kt:220]: In what scenario would BP measurements be linked without patient? Shouldn't this be deleted in the use-case where the patient is removed?

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good. Can we update the changelog for this?

File: app/src/main/java/org/simple/clinic/AppDatabase.kt (hunk 1 above)
Comment (vinaysshenoy) [reply app/src/main/java/org/simple/clinic/AppDatabase.kt:220]: Yes.

**Comment by vinaysshenoy** [comment]
> Looks good. Can we update the changelog for this?

This has been done in a later PR which will add deletions for the other resources.

**Review by Honey14 — APPROVED** [review]

**Pull request merged**